grid-proxy/tests/queries/utils.go

```diff
@@ -59,3 +59,12 @@ func SerializeFilter[F Filter](f F) string {
 
 	return res
 }
+
+func getRandomSlice[T any](slice []T) []T {
```

Comment:
Where is that used?

Reply:
I also would pass the number of elements that I need in the slice as a parameter with a guard clause. And maybe rename it to `RandomSliceFrom`